practice-app/api/urls.py

```diff
@@ -9,4 +9,6 @@
     path('eq_post/', view=views.eq_post, name='eq_post'),
     path('covid19/', view=views.covid19, name='covid19'),
     path( 'covid19/<str:countrycode>', views.covid_country, name="covid_country"),
+     path('', views.hava_api),
```

Comment:
You should add an extension to the url for your particular api. Ex: path('hava/', view=views.hava, name="hava").

Reply:
ok I changed it